### DISCUSSION TITLE
Website

### DISCUSSION THREAD

**Discussion by owenconnorz** [z-huang/music#685] [Show and tell]
You probably wouldn't like a website but I was bored and still far from done.

https://innertune.netlify.app/index.html

**Comment by owenconnorz** [comment]
Maybe if I added u to the project u might like to help when your free id like to add more images obviously for showing off the beautiful app and any things you would like just let me know

**Comment by z-huang** [comment]
GitHub is enough for now. I don't plan to create a website in the near future. However, thanks for your effort!

**Reply by owenconnorz** [reply]
No worries 👍 always here if u change your mind 😄
